persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/TreatmentMapper.xml

```diff
@@ -99,7 +99,11 @@
                     </foreach>
                 </if>
                 <if test="@java.util.Arrays@stream(studyIds.toArray()).distinct().count() > 1">
-                    (cancer_study.CANCER_STUDY_IDENTIFIER, sample.STABLE_ID) IN
+                    cancer_study.CANCER_STUDY_IDENTIFIER IN
+                    <foreach item="item" collection="uniqueStudyIds" open="(" separator="," close=")">
```

Comment:
Could you try this and see if it works
```
 <foreach item="item" collection="@java.util.Arrays@stream(studyIds.toArray()).distinct().collect(@java.util.stream.Collectors.toList())" open="(" separator="," close=")">
    #{item}
 </foreach>
```

Reply:
This also works, and this fixes Ino's concerns. Updated to match.